src/js/claims-status/components/Breadcrumbs.jsx

```diff
@@ -1,14 +1,59 @@
 import React from 'react';
+import { buildMobileBreadcrumb, debouncedToggleLinks } from '../../utils/breadcrumb-helper';
 
 class Breadcrumbs extends React.Component {
+  componentDidMount() {
+    buildMobileBreadcrumb('va-breadcrumbs-claims', 'va-breadcrumbs-claims-list');
+
+    window.addEventListener('DOMContentLoaded', () => {
+      buildMobileBreadcrumb.bind(this);
+    });
+
+    window.addEventListener('resize', () => {
+      debouncedToggleLinks('va-breadcrumbs-claims-list');
+      debouncedToggleLinks.bind(this);
+    });
+  }
+
+  componentDidUpdate() {
```

Comment:
This method may run multiple times, should we be adding new event listeners each time? It seems like adding them in `componentDidMount` would be enough.